editor/src/components/inspector/common/css-utils.ts

```diff
@@ -4746,3 +4749,163 @@ const LayoutPropertyList = [
 export function isLayoutPropDetectedInCSS(cssProps: { [key: string]: any }): boolean {
   return LayoutPropertyList.findIndex((prop: string) => cssProps[prop] != null) > -1
 }
+
+interface NonTrivialKeyword {
+  trivial: false
+}
+
+const nontrivial: NonTrivialKeyword = { trivial: false }
+
+type ParsedPropertiesWithNonTrivial = {
+  [Property in keyof ParsedProperties]: ParsedProperties[Property] | NonTrivialKeyword
+}
+
+export const trivialDefaultValues: ParsedPropertiesWithNonTrivial = {
+  // ParsedCSSProperties
+  backgroundColor: cssDefault(emptyBackgroundColor),
+  backgroundImage: [],
+  backgroundSize: [],
+  border: emptyCSSBorder,
+  borderRadius: {
+    type: 'LEFT',
+    value: {
+      value: 0,
+      unit: 'px',
+    },
+  },
+  boxShadow: [],
+  color: nontrivial,
+  fontFamily: nontrivial,
+  fontSize: nontrivial,
+  fontStyle: 'normal',
+  fontWeight: 400,
+  letterSpacing: 'normal',
+  lineHeight: 'normal',
+  mixBlendMode: 'normal',
+  opacity: nontrivial,
+  overflow: nontrivial,
+  textAlign: 'left',
+  textDecorationColor: undefined,
+  textDecorationLine: 'none',
+  textDecorationStyle: 'solid',
+  textShadow: [],
+  transform: [],
+  transformOrigin: {
+    x: CSSTransformOriginStringValueX.Center,
+    y: CSSTransformOriginStringValueY.Center,
+  },
+
+  objectFit: 'fill',
+
+  flexWrap: FlexWrap.NoWrap,
+  flexDirection: FlexDirection.Row,
+  alignItems: FlexAlignment.FlexStart,
+  alignContent: FlexAlignment.FlexStart,
+  justifyContent: FlexJustifyContent.FlexStart,
+  paddingTop: {
+    value: 0,
+    unit: 'px',
+  },
+  paddingRight: {
+    value: 0,
+    unit: 'px',
+  },
+  paddingBottom: {
+    value: 0,
+    unit: 'px',
+  },
+  paddingLeft: {
+    value: 0,
+    unit: 'px',
+  },
+  alignSelf: FlexAlignment.Auto,
+  position: 'relative',
+  left: {
+    value: 0,
+    unit: 'px',
+  },
+  top: {
+    value: 0,
+    unit: 'px',
+  },
+  right: {
+    value: 0,
+    unit: 'px',
+  },
+  bottom: {
+    value: 0,
+    unit: 'px',
+  },
+  minWidth: {
+    value: 0,
+    unit: 'px',
+  },
+  maxWidth: undefined, // should be `none`
+  minHeight: {
+    value: 0,
+    unit: 'px',
+  },
+  maxHeight: undefined, // should be `none`
+  marginTop: {
+    value: 0,
+    unit: 'px',
+  },
+  marginRight: {
+    value: 0,
+    unit: 'px',
+  },
+  marginBottom: {
+    value: 0,
+    unit: 'px',
+  },
+  marginLeft: {
+    value: 0,
+    unit: 'px',
+  },
+  flexGrow: cssUnitlessLength(0),
+  flexShrink: cssUnitlessLength(1),
+  display: 'block',
+
+  // ParsedElementProperties
+  alt: '',
+  src: '/',
+  textSizing: 'fixed',
+  ...DOMEventHandlerEmptyValues,
+  className: '',
+
+  // ParsedLayoutProperties
+  layoutSystem: undefined,
+  pinLeft: undefined,
+  pinRight: undefined,
+  centerX: undefined,
+  width: undefined,
```

Comment:
I assume this PR doesn't address this, but in case it does: what do we show if width is not explicitly set (e.g. `<span>Hello!</span>`)?

Reply:
<img width="308" alt="image" src="https://user-images.githubusercontent.com/2226774/107343392-8c4cae00-6ac1-11eb-8528-6446ccf7c8bc.png">

we see an empty Flow section

Reply:
Interesting, I thought we'd show [auto] here... but anyway, that's, out of scope!